webapp/src/components/Modal/BidConfirmationModal/BidConfirmationModal.js

```diff
@@ -48,14 +111,35 @@ export default class BidConfirmationModal extends React.PureComponent {
             values={{
               parcels: parcels.length,
               token,
-              price
+              price: formatAmount(price)
             }}
           />
         </div>
 
+        {!isLoading && balance != null ? (
+          <React.Fragment>
+            <div className="your-balance">
+              {t('auction_modal.your_balance', {
+                balance: formatAmount(balance),
+                token
+              })}
+            </div>
+            {balance < price ? (
+              <div className="insufficient-funds">
+                <Icon name="warning sign" size="small" />{' '}
+                {t('auction_modal.insufficient_funds')}
+              </div>
+            ) : null}
+          </React.Fragment>
+        ) : null}
+
         <div className="actions">
-          <Button primary onClick={this.handleSubmit}>
-            {t('auction_modal.submit')}
+          <Button
+            primary
+            onClick={this.handleSubmit}
+            disabled={price > balance || isLoading}
```

Comment:
Maybe we could move `balance < price` to a `const hasEnoughBalance = balance >= price`

Reply:
done 👍